docs/manuals/source/TasksAndConcepts/CatalogMaintenance.rst

```diff
@@ -434,6 +433,39 @@ PostgreSQL Database
    single: Database; PostgreSQL
    single: PostgreSQL
 
+.. _FreeSpacePostgres:
+
+Free space needed with PostgreSQL Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To ensure that all PostgreSQL maintenance operations like vacuuming and reindexing roll out smoothly we highly recommend to keep at least the size of your database available as free space.
```

Comment:
```suggestion
To ensure that all PostgreSQL maintenance operations like vacuuming and reindexing roll out smoothly we highly recommend not to fill the disk containing the postgresql data directory to more than 50% during normal operation. 
If this is not possible, you might need to make more space available at least temporary when the database is being upgraded to a new schema version.```

Reply:
Doesn't that change a bit the perception? Like then when I'm doing a vacuum full (normal operation) the space will be filled up, what is important is that before the operation start the space available is at least the size of the db. (in fact the size of the biggest table and indexes).